app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,4 +1,4 @@
-$primary: #1a454a;
+$primary: #043c54;
```

Comment:
In other applications, we have the primary set to #1A454A

Reply:
Oh I see. I'm not sure if we should be using that or the BN "official" colour.
@farhatahmad What do you think?

Reply:
Leave it as #1a454a for now - we'll need to change it to blue later anyways